### PR TITLE
Documenting bit shifting operations

### PR DESCRIPTION
**Pull request by iFrostizz**
Addings docs for `cast shr / shl` from PR: https://github.com/foundry-rs/foundry/pull/2416

### CODE REVIEW

**Comment by iFrostizz** [comment]
Should I mention that we can either use `10` or `dec` for the bases or is it explicit enough ?

File: src/reference/common/shift-options.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+#### Base option
```
Comment (onbjerg) [inline src/reference/common/shift-options.md:1]:
```suggestion
#### Base Options
```

File: src/reference/common/shift-options.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+
+`--base-in` *dec* / *hex*
+
+&nbsp;&nbsp;&nbsp;&nbsp;The base of the number that is in.
+
+`--base-out` *dec* / *hex*
+
+&nbsp;&nbsp;&nbsp;&nbsp;The base of the number that is out.
```
Comment (onbjerg) [inline src/reference/common/shift-options.md:9]:
```suggestion
`--base-in` *dec* / *hex*  
&nbsp;&nbsp;&nbsp;&nbsp;The base of the input number (10, 16, dec, or hex)

`--base-out` *dec* / *hex*  
&nbsp;&nbsp;&nbsp;&nbsp;The desired base of the output (10, 16, dec, or hex)
```

**Review by onbjerg — APPROVED** [review]
small nits

**Comment by onbjerg** [comment]
@all-contributors add @iFrostizz for docs

**Pull request merged**